index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Ali Aqa Atayee</title>
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&display=swap" rel="stylesheet">
 </head>
+
 <body>
-   
+    <div class="home-page">
+        <header>
+            <nav id="nav">
+                <a class="logo" href="#"><img src="assets/images/logo-dark.png" alt="logo" /></a>
+                <nav class="hamburger"><img src="assets/images/Union.svg" alt="menu"></nav>
+                <ul class="menu-link">
+                    <li><a href="">Projects</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href=""> Contact </a></li>
+                </ul>
+            </nav>
+        </header>
+        <section id="banner">
+            <h4 class="slogan">Hey there. I’m <br>Ali <br>
+                <span>I’m a software developer</span>
+            </h4>

```

Comment:
- [ ] Hi your design matches nicely in this section. though in this section, could you use a `h1` tag because that signifies it as the main heading on the page.  A `h4` should be used for other lower-ranking headings.
- [ ] The text in the `span` should be in a `p` tag. it's more of a description that is a much longer piece of text.

Reply:
Hi dear @MrBrN197, thank you for your precious feedback and comments sure I will modify it